documentation/modules/ROOT/pages/modules/cdc-mysql-connector/c_how-the-mysql-connector-performs-database-snapshots.adoc

```diff
@@ -1,7 +1,6 @@
 // Metadata created by nebel
 //
 :isImageReady: false
-:imagesdir: /assets/images
```

Comment:
This is probably the only change I see that may raise a concern, particularly for downstream consumption of the docs as I understand the {{_attributes.adoc}} file isn't consumed by that build process; so this _might_ introduce a small incompatibility with downstream.  @fbolton can you let us know if this breaks something downstream as time permits?  If it does, perhaps we can find a better way to handle this for you guy.

Reply:
IIRC, @fbolton also was advocating for removing those attributes from individual files to avoid the redundancy.

Reply:
To solve it in the source, one could use ifdef macros. If this is a common attribute downstream, it could be for example a command line argument it is uses plain Asciidoctor.